subsys/bluetooth/shell/ll.c

```diff
@@ -258,9 +257,22 @@ int cmd_advx(const struct shell *shell, size_t argc, char *argv[])
 		goto exit;
 	}
 
+#if defined(CONFIG_BT_LL_SW_SPLIT)
+	printk("ad data set...");
```

Comment:
Is it appropriate for a shell module to use printk? Isn't the recommendation to use the shell's own output APIs?

Reply:
This is an oversight, will fix it.